### PR TITLE
Fix PHCPS errors

### PR DESCRIPTION
**Pull request by ethitter**
Depends on #94

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,18 +9,68 @@ branches:
...
-    - 5.5
-    - 5.6
-    - 7.0
-
-env:
-    - WP_VERSION=nightly WP_MULTISITE=0
-    - WP_VERSION=nightly WP_MULTISITE=1
-    - WP_VERSION=4.4 WP_MULTISITE=0
-    - WP_VERSION=4.4 WP_MULTISITE=0
+matrix:
```
Comment (timatron) [inline .travis.yml]: no way to make this less verbose?

File: .travis.yml [hunk 2]
```diff
@@ -32,9 +82,9 @@ before_script:
       composer global require "phpunit/phpunit=4.8.*"
     fi
   - |
-    composer global require wp-coding-standards/wpcs
```
Comment (timatron) [inline .travis.yml]: sure we dont need to require `wp-coding-standards/wpcs`?

**Comment by timatron** [comment]
🦅

File: .travis.yml (hunk 1 above)
Comment (ethitter) [reply .travis.yml]: Only by dropping 5.3 support. It's a bug in Travis, as 5.3 requires the `precise` distro. https://github.com/travis-ci/travis-yaml/issues/118

File: .travis.yml (hunk 2 above)
Comment (ethitter) [reply .travis.yml]: Yes, it's a dependency of VIPWPCS.

**Pull request merged**